src/components/TaskCard/index.module.scss

```
@@ -13,6 +13,20 @@
   border: 0.063rem solid $lilac-blue;
   border-radius: 0.313rem;
 }
+.actionButtonContainer {
```

use line break to separate style rules.

```suggestion
}

.actionButtonContainer {
```